### PR TITLE
fix connectivity yaml for arm32

### PR DESCRIPTION
**Pull request by philipktlin**
fix connectivity yaml for arm32

### CODE REVIEW

File: builds/e2e/connectivity.yaml [hunk 1]
```diff
@@ -177,14 +209,30 @@ jobs:
...
+        condition: and(succeeded(), eq(variables['run.steps'], 1))
       - task: AzureKeyVault@1
+        condition: and(succeeded(), eq(variables['run.steps'], 1))
         displayName: 'Azure Key Vault'
         inputs:
           azureSubscription: $(azure.subscription)
           KeyVaultName: $(azure.keyVault)
           SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,EdgeConnectivityTestHubConnString,EdgeConnectivityEventHubConnString,kvLogAnalyticWorkspaceId,kvLogAnalyticSharedKey,EdgeConnectivityStorageAccountConnString'
       - task: DownloadBuildArtifacts@0
+        condition: and(succeeded(), eq(variables['run.steps'], 1))
```
Comment (yophilav) [inline builds/e2e/connectivity.yaml]: By default, `succeeded()` is a combination of `Succeeded` and `SucceededWithIssues` (continued w/ error)

Are you sure that is what you want there?

File: builds/e2e/connectivity.yaml [hunk 2]
```diff
@@ -195,6 +243,7 @@ jobs:
           downloadPath: '$(Build.StagingDirectory)'
           artifactName: $(edgelet.artifact.name)
       - task: DownloadBuildArtifacts@0
+        condition: and(succeeded(), eq(variables['run.steps'], 1))
```
Comment (yophilav) [inline builds/e2e/connectivity.yaml]: Where did `run.steps` get set? I didn't see it in the built-in DevOps document.

Comment (yophilav) [reply builds/e2e/connectivity.yaml]: Please rename the var.

File: builds/e2e/templates/set-run-flag.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+steps:
+  - task: Bash@3
+    name: Set_run_flag
```
Comment (yophilav) [inline builds/e2e/templates/set-run-flag.yaml]: Please add more description that this block of code is for job cancellation.

File: builds/e2e/connectivity.yaml (hunk 1 above)
Comment (philipktlin) [reply builds/e2e/connectivity.yaml]: this is exactly what we want as succeeded() will ensure all previous tasks succeeded.

**Pull request merged**